server/chat.ts

```diff
@@ -740,7 +749,8 @@ export class CommandContext extends MessageContext {
 		if (this.room) this.room.roomlog(data);
 	}
 	addModAction(msg: string) {
-		this.room!.addByUser(this.user, msg);
+		if (this.pmTarget) return Chat.sendPM(`/log ${msg}`, this.user, this.pmTarget);
+		if (this.room) return this.room.addByUser(this.user, msg);
```

Comment:
We should probably have a `context.addByUser` function that abstracts this away.

Reply:
(It turned out that this was low-level enough that it was best done a different way, namely f16849eda39a306a88)